### PR TITLE
Import and add in-progress release notes.

### LINKED ISSUE #1155: UI/UX for in-progress release notes
This URL in the old website:

https://www.boost.org/users/history/in_progress.html

displays the in-progress release notes for the next Boost version. The current redirection rules for the new website map

https://boost.io/users/history/in_progress.html

to

https://www.boost.io/releases/boost-in-progress/

which is a 404.

This need yet to be specified. Some issues to consider that are potential blockers/difficulties:

* If we add "in_progress" to the Releases drop-down list, we'll get a mismatch when switching to "Libraries".
* I think the in-progress release notes are not being perodiocallly (like, daily) copied to the new website.

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1155
- add in-progress release notes import to the import_release_notes task

### CODE REVIEW

**Comment by brianjp93** [comment]
image

- outstanding question of how users can visit this page. Should there be a link from the releases page that says "view in-progress release notes"?

File: templates/versions/in_progress_release_notes.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <!-- alert for non-current Boost versions -->
+    {% include "libraries/includes/version_alert.html" %}
+
+    <section id="libraryReadMe"
+      class="p-6 mb-4 bg-white md:rounded-lg md:shadow-lg dark:text-white text-slate dark:bg-charcoal dark:bg-neutral-700">
+      {% if release_notes %}
+        {{ release_notes|safe }}
+      {% else %}
+        No data for in-progress release notes yet. Come back soon!
+      {% endif %}
```
Comment (GregKaleka) [inline templates/versions/in_progress_release_notes.html:21]: This is of course fine, but could be simplified to:

```suggestion
        {{ release_notes|safe|default:"No data for in-progress release notes yet. Come back soon!" }}
```

File: versions/views.py [hunk 2]
```diff
@@ -154,3 +155,25 @@ def get_object(self, queryset=None):
...
+class InProgressReleaseNotesView(TemplateView):
+    template_name = "versions/in_progress_release_notes.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update(
+            {
+                "release_notes": self.get_in_progress_release_notes(),
+            }
+        )
```
Comment (GregKaleka) [inline versions/views.py:169]: I think I would just use the simpler

```suggestion
        context["release_notes"] = self.get_in_progress_release_notes()
```

**Review by GregKaleka — APPROVED** [review]
LGTM - approving, with two optional simplification suggestions.

**Comment by GregKaleka** [comment]
> outstanding question of how users can visit this page. Should there be a link from the releases page that says "view in-progress release notes"? 

Yes, that makes sense to me, but I'll let others chime in.

**Comment by brianjp93** [comment]
@rbbeeston I would prefer to not put the "in-progress" option in the version dropdown and would rather opt for a link elsewhere on the release page.
For all other options in the dropdown, selecting a version sets it for the user in a cookie so that it persists when they come back or go to the libraries list page.  The "in progress" option however, cannot persist as it is not an actual version, and doesn't mean anything for the library list page.
What do you think?

**Comment by rbbeeston** [comment]
There seems to be some pages that were in in progress that weren't actually used, so perhaps there is an opportunity there to do something with this and those other pages that will allow for that.

I think just get this wrapped up as is, and we will have a separate issue for hooking it up. So this one can be wrapped up.

**Pull request merged**